packages/appstore/src/components/pre-loader/regulations-switcher-loader.tsx

```diff
@@ -1,21 +1,24 @@
-import ContentLoader from 'react-content-loader';
 import React from 'react';
-import { DesktopWrapper, MobileWrapper } from '@deriv/components';
+import ContentLoader from 'react-content-loader';
+import { useDevice } from '@deriv-com/ui';
+
+const RegulationsSwitcherLoader = () => {
+    const { isDesktop } = useDevice();
 
-const RegulationsSwitcherLoader = () => (
-    <ContentLoader
-        backgroundColor={'var(--general-section-1)'}
-        foregroundColor={'var(--general-hover)'}
-        width={300}
-        height={60}
-    >
-        <DesktopWrapper>
-            <rect x='10' y='10' rx='6' ry='6' width='190' height='40' />
-        </DesktopWrapper>
-        <MobileWrapper>
-            <rect x='60' y='10' rx='6' ry='6' width='80' height='30' />
-        </MobileWrapper>
-    </ContentLoader>
-);
+    return (
+        <ContentLoader
+            backgroundColor={'var(--general-section-1)'}
```

Comment:
Will `backgroundColor='var(--general-section-1)'` work?

Reply:
@kate-deriv I'm sure it will :)